### PR TITLE
ioplus instead of kuio

### PR DESCRIPTION
**Pull request by Yoti**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -30,11 +30,13 @@ add_executable(TurboPad
...
-  SceAppMgr_stub
-  SceDisplay_stub
-  kuio_stub
+  SceLibKernel_stub_weak
+  SceCtrl_stub_weak
+  SceIofilemgr_stub_weak
+  SceAppMgr_stub_weak
+##SceTouch_stub_weak
+  SceDisplay_stub_weak
+##SceSysmem_stub_weak
```
Comment (Rinnegatamante) [inline CMakeLists.txt]: what are these changes?

**Review by Rinnegatamante — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (Yoti) [reply CMakeLists.txt]: I got this from remaPSV.

Comment (Rinnegatamante) [reply CMakeLists.txt]: Can you remove the commented ones?

Comment (Yoti) [reply CMakeLists.txt]: Done.

**Pull request merged**